src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,74 @@
+.calendar {
+  height: 100vh;
```

Comment:
It's not working as expected - you forgot to remove default margins from `h1` tags. Check it in dev tools.

Reply:
Yep, already fixed that, and used that property for the block .calendar__container for centering